docs/src/apalache/antipatterns.md

```diff
@@ -29,7 +29,7 @@ min(S) ==
     IN IF e < minOther THEN e ELSE minOther 
 ```
 
-Apalache dislikes the use of the above, for several reasons. Firstly, since the operator is `RECURSIVE`, Apalache requires a predefined upper bound on unrolling, which means that the user must know, ahead of time, what the largest `|S|` is, for any set `S`, to which this operator is ever applied. 
+Apalache dislikes the use of the above, for several reasons. Firstly, since the operator is `RECURSIVE`, Apalache would not support it after 0.23.1. In earlier versions Apalache requires a predefined upper bound on unrolling, which means that the user must know, ahead of time, what the largest `|S|` is, for any set `S`, to which this operator is ever applied. 
```

Comment:
```suggestion
Apalache dislikes the use of the above, for several reasons. Firstly, since the operator is `RECURSIVE`, does not support it after 0.23.1. In earlier versions Apalache requires a predefined upper bound on unrolling, which means that the user must know, ahead of time, what the largest `|S|` is, for any set `S`, to which this operator is ever applied. 
```